instructure/canvas-ios
# Offline sync dashboard notification

refs: MBL-16649
affects: Student
release note: none

test plan:
- Start an offlline sync.
- Observe the dashboard notification about the sync progress.
- Try using the dismiss button.
- Check voiceover.

## Screencast

https://github.com/instructure/canvas-ios/assets/72396990/0d408ff0-b32f-40c2-af81-aa975cbe56b4

## Checklist
